docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,65 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using `--device` option
+with docker run. An example of passing a device with docker is given below:
+```
+$ sudo docker run -it --device=/dev/sdb debian
+```
+Clear Containers has support for passing the following devices to the
+container with `--device`.
+
+## Virtual Function I/O (VFIO) Devices (PCI Passthrough)
+
+The VFIO driver is a framework for exposing direct device access to userspace, 
```

Comment:
Lines 13-14 rewrite: "The VFIO driver is a framework that exposes direct device access to userspace in a secure, Input-Output..."

Reply:
Picked verbatim from kernel docs.